### PR TITLE
"contact section for homepage tested"

### PR DESCRIPTION
**Pull request by richardcolachagua**
The first branch I made, I could not get the homepage.tsx regardless of pulling the latest from main. I made a new branch and got the render to show on the homepage.

### CODE REVIEW

File: src/pages/HomePage/ContactSection.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  Grid,
+  useTheme,
+  Stack,
+} from "@mui/material";
+import EmailIcon from "@mui/icons-material/Email";
+import { FaInstagram } from "react-icons/fa";
+
+const ContactSection = () => {
+  const theme = useTheme();
+  const SPACING = 2;
```
Comment (AlanisYates) [inline src/pages/HomePage/ContactSection.jsx:18]: This is not needed - we shouldnt hard code theme varibales like this. you can put it in the component.

File: src/pages/HomePage/ContactSection.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from "react";
```
Comment (AlanisYates) [inline src/pages/HomePage/ContactSection.jsx:1]: Remove unused Props

File: src/pages/HomePage/ContactSection.jsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+            justifyContent: "space-between",
+          }}
+        >
+          <Link href="/TOS" sx={{ textDecoration: "none", color: "white" }}>
+            Terms & Conditions
+          </Link>
+          <Typography variant="body2">
+            © 2023 The Sneaker Society - All rights reserved
+          </Typography>
+        </Stack>
```
Comment (AlanisYates) [inline src/pages/HomePage/ContactSection.jsx:143]: This should be a footer and suck at the bottom takwe a look at how it looks on mobile

File: src/pages/HomePage/ContactSection.jsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+  const SPACING = 2;
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: "black",
+        color: "white",
+        padding: theme.spacing(8, 0),
+        textAlign: "center",
+      }}
```
Comment (AlanisYates) [inline src/pages/HomePage/ContactSection.jsx:27]: This Does not seem to be a full page

File: src/pages/HomePage/ContactSection.jsx [hunk 5]
```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from "react";
+import {
+  Typography,
+  Box,
+  Link,
+  IconButton,
+  Divider,
```
Comment (AlanisYates) [inline src/pages/HomePage/ContactSection.jsx:7]: Remove unused imports

File: src/pages/HomePage/ContactSection.jsx [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+  Container,
+  Grid,
+  useTheme,
+  Stack,
+} from "@mui/material";
+import EmailIcon from "@mui/icons-material/Email";
+import { FaInstagram } from "react-icons/fa";
```
Comment (AlanisYates) [inline src/pages/HomePage/ContactSection.jsx:14]: You need to add react icons to the package.json

**Review by AlanisYates — CHANGES_REQUESTED** [review]
Good work almost there! Also try adding this extension in your browser to test it out in mobile itss still looking weird on my end https://chromewebstore.google.com/detail/mobile-simulator-responsi/ckejmhbmlajgoklhgbapkiccekfoccmk?pli=1

File: src/pages/HomePage/ContactSection.jsx (hunk 5 above)
Comment (richardcolachagua) [reply src/pages/HomePage/ContactSection.jsx:7]: removed unused imports

File: src/pages/HomePage/ContactSection.jsx (hunk 6 above)
Comment (richardcolachagua) [reply src/pages/HomePage/ContactSection.jsx:14]: added react icons

File: src/pages/HomePage/ContactSection.jsx (hunk 2 above)
Comment (richardcolachagua) [reply src/pages/HomePage/ContactSection.jsx:1]: removed unused props

File: src/pages/HomePage/ContactSection.jsx (hunk 1 above)
Comment (richardcolachagua) [reply src/pages/HomePage/ContactSection.jsx:18]: removed

File: src/pages/HomePage/ContactSection.jsx (hunk 3 above)
Comment (richardcolachagua) [reply src/pages/HomePage/ContactSection.jsx:143]: updated for mobile

**Comment by AlanisYates** [comment]
@richardcolachagua Take alook at the desktop version for me it looks like this
mobile
Also you need to makes sure the footer is at the bottom

File: src/pages/HomePage/ContactSection.jsx [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+const ContactSection = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: "black",
+        height: "150vh",
```
Comment (AlanisYates) [inline src/pages/HomePage/ContactSection.jsx:18]: Why is this 150?

Comment (richardcolachagua) [reply src/pages/HomePage/ContactSection.jsx:18]: removed, was actually serving no purpose

**Review by AlanisYates — APPROVED** [review]

**Pull request merged**